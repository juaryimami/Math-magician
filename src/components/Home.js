const Home = () => (
  <div className="home_container">
    <h2 data-testid="home-1" className="home_title"> welcome to our page</h2>
    <p className="home_paragraph">
      {' '}
      fsols skfjslfjs lsjslj sjl sljfls lsfjljsljsljslj lsjfls lfjslfj jslj jslj
      lsjlsljls sjsljs sj sj sljsljslj jsfjslj sjfsl sjljsl
      {' '}
    </p>
  </div>
);

export default Home;
