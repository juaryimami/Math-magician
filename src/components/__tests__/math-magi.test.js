import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('renders home page correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('test', () => {
  expect(true).toBe(true);
});
