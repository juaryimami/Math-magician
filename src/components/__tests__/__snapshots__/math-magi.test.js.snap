// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page correctly 1`] = `
<div
  className="home_container"
>
  <h2
    className="home_title"
    data-testid="home-1"
  >
     welcome to our page
  </h2>
  <p
    className="home_paragraph"
  >
     
    fsols skfjslfjs lsjslj sjl sljfls lsfjljsljsljslj lsjfls lfjslfj jslj jslj lsjlsljls sjsljs sj sj sljsljslj jsfjslj sjfsl sjljsl
     
  </p>
</div>
`;
